source/features/github-bugs.css

```diff
@@ -10,16 +10,19 @@
 	margin: -5px 0;
 }
 
-/* Reposition committer's avatar on PR commits #3444 */
-.pull-request-tab-content .commit .AvatarStack {
-	display: inline-block;
-	margin-bottom: -4px;
-	margin-right: -4px;
-}
-
 /* Add missing border radius to collapsed PR files #3444 */
 .js-blob-wrapper,
 .file:not(.open) .file-header.file-header--expandable {
 	border-bottom-left-radius: 5px;
 	border-bottom-right-radius: 5px;
 }
+
+/* Align author name on PR sticky header */
+.sticky-content .author.css-truncate-target {
+	vertical-align: -4px;
+}
+
+/* Align dropdown button on file page */
+#blob-more-options-details {
+	margin-top: 5.6px;
```

Comment:
This is to align the dropdown button (whose height is `20.8px`) with the find file button (whose height is `32px`)

(32 - 20.8) / 2 = 5.6

Reply:
There's `display: flex` on that line, just add `align-items: center !important;` on `.d-flex` to align the whole line since it's not aligned:

<img width="750" alt="" src="https://user-images.githubusercontent.com/1402241/96290266-ea520480-0fab-11eb-9a3d-34c7f5168efe.png">

Reply:
I hope this is fixed across the board, the whole bar is probably the most broken-looking part of GitHub/Refined GitHub. Example: https://github.com/sindresorhus/refined-github/issues/3671 (not totally RG’s fault, but every element has its own alignment)